meta-balena-common/recipes-core/balena-rollback/files/rollback-tests

```diff
@@ -36,6 +36,10 @@ fi
 # Balena health check
 if /usr/lib/balena/balena-healthcheck ; then
 	echo "Rollback: Balena looks healthy"
+	echo "Rollback: Cleaning up storage migration..."
+	env BALENA_A2O_STORAGE_ROOT="/mnt/data/docker" a2o-migrate -debug -cleanup
```

Comment:
I see cleanup. But not really a test if overlay is working?
Also, checked a2o-migrate cleanup, I'd recommend there should be a basic sanity check if the overlay directory is present or not.